src/main/java/uk/gov/hmcts/darts/log/service/CasesLoggerService.java

```diff
@@ -9,4 +9,6 @@ public interface CasesLoggerService {
     void defendantNameOverflow(AddCaseRequest addCaseRequest);
 
     void defendantNotAdded(String defendant, String caseNumber);
+
+    void caseDeletedDueToExpiry(Integer caseId, String caseNumber);
```

Comment:
Could you expose this to other packages via the `LogApi` class please. We have a pattern to use an "api" package for methods that are used by other packages. Allowing us to easily see all usages of a method outside the package it's defined within.

Reply:
done